docs/content/customization/aws/identity-ref.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Identity Reference"
++++
+
+The identity reference customization allows the user to specify the AWS identity to use when reconciling the cluster.
+This identity reference can be used to authenticate with AWS services using different identity types such as
+AWSClusterControllerIdentity, AWSClusterRoleIdentity, or AWSClusterStaticIdentity.
```

Comment:
We can link them to public CRD definitions

Reply:
Yes its linked a few lines below